https://github.com/MariaDB/server/pull/157

MDEV-9657 - Use /bin/sh #157 

Sp1l commented Feb 27, 2016

Remove Linuxism.

Works fine with a POSIX-compat shell

--- storage/tokudb/PerconaFT/buildheader/CMakeLists.txt.orig	2016-01-28 12:12:55 UTC
+++ storage/tokudb/PerconaFT/buildheader/CMakeLists.txt
@@ -1,6 +1,6 @@
 set_directory_properties(PROPERTIES INCLUDE_DIRECTORIES "")
 
-file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/runcat.sh" "#!/bin/bash
+file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/runcat.sh" "#!/bin/sh
 out=$1; shift
 exec \"$@\" >$out")
 
@@ -8,7 +8,7 @@ add_executable(make_tdb make_tdb.cc)
 set_property(TARGET make_tdb APPEND PROPERTY COMPILE_DEFINITIONS _GNU_SOURCE)
 add_custom_command(
   OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/db.h"
-  COMMAND bash runcat.sh "${CMAKE_CURRENT_BINARY_DIR}/db.h" $<TARGET_FILE:make_tdb>
+  COMMAND sh runcat.sh "${CMAKE_CURRENT_BINARY_DIR}/db.h" $<TARGET_FILE:make_tdb>
   DEPENDS make_tdb)
 add_custom_target(install_tdb_h DEPENDS
   "${CMAKE_CURRENT_BINARY_DIR}/db.h")
